app/models/pacbio/library.rb

```diff
@@ -101,5 +101,17 @@ def check_for_associated_wells
         throw(:abort)
       end
     end
+
+    def check_for_associated_pools
```

Comment:
Inwould think about turning this into a scope or a method in aliquot. That way everything is delegated to aliquot and you just need to return a boolean.

Reply:
👍